src/polychord/utils.F90

```diff
@@ -14,6 +14,9 @@ module utils_module
     !> \f$ \log(2\pi) \f$ in real(dp)
     real(dp), parameter :: logTwoPi = log(8d0*atan(1d0))
 
+    !> The threshold for 'log(0) values'
+    real(dp), parameter :: logzero = -1d30
+
```

Comment:
This should definitely be removed, as logzero is now something that's stored in settings, and this will shadow that/cause general confusion

Reply:
I can remove it again but that breaks the examples.

Reply:
Ah, I assume by this you mean the likelihood/examples/fitting.f90 and likelihoods/examples/gaussian_shells.f90 (having done a quick grep for logzero). I have just pushed a fix to master, so if you merge in the latest changes this should work.